resources/public/api/conf/5.0/schemas/foreign_property_cumulative_summary_retrieve/def1/response.json

```diff
@@ -0,0 +1,198 @@
+                "multipleOf": 0.01,
+                "minimum": -99999999999.99,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "travelCosts": {
+                "type": "number",
+                "description": "Car, van and travel costs incurred in running a property business. The value must be between -99,999,999,999.99 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": -99999999999.99,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "costOfServices": {
+                "type": "number",
+                "description": "Cost of services provided, including wages. The value must be between -99,999,999,999.99 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": -99999999999.99,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "residentialFinancialCost": {
+                "type": "number",
+                "description": "The residential financial cost deductible from rental income (tax relief). The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": 0.00,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "broughtFwdResidentialFinancialCost": {
+                "type": "number",
+                "description": "Amount of relief brought forward for restricted residential financial costs. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": 0.00,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "other": {
+                "type": "number",
+                "description": "Other allowable property expenses. The value must be between -99,999,999,999.99 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": -99999999999.99,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              },
+              "consolidatedExpenses": {
+                "type": "number",
+                "description": "The sum of all allowable expenses for the specified period.The value must be between -99,999,999,999.99 and 99999999999.99 up to 2 decimal places.",
+                "multipleOf": 0.01,
+                "minimum": -99999999999.99,
+                "maximum": 99999999999.99,
+                "example": 5000.99
+              }
+            },
+            "additionalProperties": false
+          }
+        },
+        "required": [
+          "countryCode"
+        ],
+        "additionalProperties": false
+      },
+      "required": ["foreignPropertyEntry"]
```

Comment:
This can be removed since there is technically no field called `foreignPropertyEntry`. Please also remove the comma above once it has been taken out, thanks